### PR TITLE
Bad url for documentation

### PR DESCRIPTION
**Pull request by lipki**

### CODE REVIEW

File: Documentation/NextSteps/Index.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Next Steps and Further Reading
 ==============================
 
 All of TYPO3's documentation is available 
-`docs.typo3.org <https://docs.typo3.org/typo3cms/>`_.
+`docs.typo3.org <https://docs.typo3.org/>`_.
```
Comment (mbrodala) [inline Documentation/NextSteps/Index.rst:11]: Can you also update all other similar links here, too?

Comment (lipki) [reply Documentation/NextSteps/Index.rst:11]: maybe

Comment (lipki) [reply Documentation/NextSteps/Index.rst:11]: In this branch I can only find this one.

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
Backported to 9.5, already fixed in master

**Pull request merged**